docs/content/anonymization/core-anonymizers.md

```diff
@@ -265,6 +265,87 @@ customer:
 ```
 :::
 
+## NullAnonymizer
+
+Set all value to `NULL`.
+
+::: code-group
+```php [Attribute]
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use MakinaCorpus\DbToolsBundle\Attribute\Anonymize;
+
+#[ORM\Entity()]
+#[ORM\Table(name: 'customer')]
+class Customer
+{
+    // ...
+
+    #[ORM\Column]
+    #[Anonymize(type: 'null')] // [!code ++]
+    private ?string $sensibleContent = null;
+
+    // ...
+}
+```
+
+```yml [YAML]
+# config/anonymization.yaml
+
+customer:
+    sensible_content: 'null'
+
+#...
+```
+:::
+
+## ConstantAnonymizer
+
+Set all value to `xxxxxx` or a custom value  (with the `value` option).
```

Comment:
```suggestion
Set all values to `xxxxxx` or a custom value  (with the `value` option).
```
I wonder if this default value is really pertinent. What if we make developers always define a value?

Reply:
Yep, it was more pertinent when this anonymizer name was `BlankingAnonymizer`, like a marker stroke. But with this name, it makes no sense, you are right. I'm gonna update this